src/main/java/group1/intern/controller/customer/PayController.java

```diff
@@ -0,0 +1,30 @@
+package group1.intern.controller.customer;
+
+import group1.intern.bean.PayForm;
+import jakarta.validation.Valid;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PostMapping;
+
+@Controller
+public class PayController {
+    @GetMapping("/payments")
+    public String showPaymentPage(Model model) {
+        model.addAttribute("payForm", new PayForm());
+        return "screens/payment/pay";
```

Comment:
```suggestion
        return "screens/payments/index";
```
cái này là list payment hay là 1 payment em nhỉ? list thì tên html là index, còn 1 object thì show nhá
ngoài ra, tên folder payments là số nhiều nhé 

thêm: khi hiển thì ra màn hình để create payment thì thông tin user như name, địa chỉ ... thì mình nên hiển thị default lấy từ DB là đẹp, sửa hay ko là quyền của khách hàng nhá